src/helpers/git/get-git-data.ts

```diff
@@ -54,10 +54,22 @@ export const gitTrackedFiles = async (git: simpleGit.SimpleGit): Promise<string[
 
 export const gitBranch = async (git: simpleGit.SimpleGit): Promise<BranchSummary> => git.branch()
 
+export const gitCurrentBranch = async (git: simpleGit.SimpleGit): Promise<string> => {
+  const branch = await git.raw(['branch', '--show-current'])
+
+  return branch.trim()
+}
+
 export const gitMessage = async (git: simpleGit.SimpleGit): Promise<string> => git.show(['-s', '--format=%s'])
 
 export const gitAuthorAndCommitter = async (git: simpleGit.SimpleGit): Promise<string> =>
   git.show(['-s', '--format=%an,%ae,%aI,%cn,%ce,%cI'])
 
 export const gitRepositoryURL = async (git: simpleGit.SimpleGit): Promise<string> =>
   git.listRemote(['--get-url']).then((url) => url.trim())
+
+export const gitLocalCommitShas = async (git: simpleGit.SimpleGit, branchName: string): Promise<string[]> => {
+  const gitShas = await git.raw(['log', branchName, '--not', '--remotes', '--format=%H'])
```

Comment:
Do we have a limit to pass here, or when does the `log` stops in this case?

Reply:
Good point. I think this has no limits. Let me check

Reply:
Indeed it is unlimited, I will add a limit of 10 commits